### ISSUE TITLE
Project Infra: Oracle Cloud Access to Deploy Visualizations of OTEL Project Statistics

### ISSUE THREAD

**Issue by adrielp** [open-telemetry/community#2342]
The ask came up in the Semantic Conventions SIG call to leverage OTEL Collector components that observe GitHub within the OTEL projects. After discussing with @austinlparker  & @trask  in the infra SIG, it was asked to open this issue. 

The request is to get enough access to run an OTEL collector, observability backend, and visualization layer. Eventually we'd like to make the dashboard public after locking things down correctly. 

It can show things like: 

* repository health metrics
* metrics from events
* eventually traces from workflows run within in the project.

**Comment by austinlparker** [comment]
Created a compartment and user group for this in OCI. Just need to create users and assign them.

**Comment by adrielp** [comment]
Yes! I have the access needed at this point in time and have started developing. Closing this issue, thanks!
